ArrayOps.java

```diff
@@ -1,26 +1,101 @@
 public class ArrayOps {
     public static void main(String[] args) {
-        
+    // System.out.println(findMissingInt(new int[] {1, 2}));
+    System.out.println(secondMaxValue(new int[] {1, -2, 3, -4, 5}));
+    // containsTheSameElements(new int[] {1, 5}, new int[] {5, 1, 5, 1, 5});
+    // isSorted(new int[] {1, -2, 3});
     }
     
+    
     public static int findMissingInt (int [] array) {
-        // Write your code here:
-        return -1;
+        int missingInt = -1;
+        boolean isMissing = false;
+        for (int i = 0; i < array.length + 1; i++) {    
+            for (int arrayIndex = 0; arrayIndex < array.length; arrayIndex++) {
+                /* Check if the value is in the array, if yes then
+                 * continue to the next value in given array.
+                 */
+                if ((i != array[arrayIndex])) {
+                    isMissing = true;
+                    missingInt = i;
+                } else if (i == array[arrayIndex]) {
+                    isMissing = false;
+                    break;
+                }
```

Comment:
```suggestion
               if (i == array[arrayIndex]) {
                    isMissing = false;
                    break;
                }
```
No need the first if, with your loop you might realize the missing item only after you iterate over all items - so it better to break when it found and mention that next item should be looking for.